woocommerce/wc-admin
# Update verified product owner view for a review

Fixes #5862 

Moved the verified owner checkmark up to the title, had to add an svg component for the checkmark icon, as this type  wasn't
available in the gridicons and wordpress/icons.

### Screenshots

<img width="510" alt="Screen Shot 2020-12-14 at 4 49 01 PM" src="https://user-images.githubusercontent.com/2240960/102134480-f41daa00-3e2c-11eb-9579-d705bb8e0453.png">

### Detailed test instructions:

- Create a store with product reviews enabled
- Create a completed order
- Write a product review with the same email address used in the completed order (Load in incognito window to input email address on review)
- Green checkmark should now appear in the home screen review panel for the one just written
- On hover over the checkmark, it should show a tooltip with `Verified owner`.
- Write another review with another email
- Checkmark should not be present
